src/components/Home/VoteList.tsx

```diff
@@ -42,7 +33,6 @@ const VoteList = () => {
   };
 
   const getMoreItem = async () => {
-    setIsLoaded(true);
     const newData = await getCurrentVoteData(Number(CursorId));
 
     if (!dataList) {
```

Comment:
내가 이거 수정해서 master에 푸쉬했는데 이게 데이터 호출말고 타입 변환과정에서 result에 대해 undefined가 뜨더라고?!
그래서 타입 좁히기를 해주어야 할것같아!

```suggestion
 const getMoreItem = useCallback(async () => {
    const newData = await getCurrentVoteData(Number(cursorId));
    if (newData?.data) {
      const newDataList = newData.data.result as VoteCardInfo[];
      setDataList(dataList.concat(newDataList));
      setCursorId(newData.data.resCursorId);
    }
  }, [cursorId]);
```

Reply:
오 타입 좁히기는 처음 접해본다!!!
덕분에 아티클 찾아보면서 공부했어용 ㅎㅎㅎ
DataList가 처음에 undefined니까 조건문 안에서 호출된 데이터를 변환할 때 undefined로 추론해버리는건가...?!!!